### PR TITLE
Update README to include instructions for first time

### PR DESCRIPTION
**Pull request by Ian-Nicholls**
**WHAT**

The README is missing some steps when doing this for the first time

**WHY**

Without the updated instructions the page doesn't have the correct CSS and looks very meh and an error at the top

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,11 @@ This is the product page for GovWifi. It is a [middleman site](http://middlemana
 
 ## Getting Setup for Developement
```
Comment (joelanman) [inline README.md:5]: can we fix the old typo in developement too? :)

File: README.md [hunk 2]
```diff
@@ -4,7 +4,11 @@ This is the product page for GovWifi. It is a [middleman site](http://middlemana
...
 
-The recommended ruby version is `2.7.2`. You may need to install this.
+Packages required to be installed to get the local version working are:
```
Comment (joelanman) [inline README.md:7]: maybe just 'Software' instead of 'Packages' - I think its only called packages in linux

File: README.md [hunk 3]
```diff
@@ -19,9 +23,12 @@ You can now open the codebase in any text editor and make changes.
 
 To make the site viewable in your browser:
 
+- npm install
 - `make serve`
 - Open your browser and go to http://localhost:4567
 
+All changes now made to the pages you've checked out are visible with a simple browser refresh (F5)
```
Comment (joelanman) [inline README.md:30]: is this already on line 34?

File: README.md (hunk 1 above)
Comment (Ian-Nicholls) [reply README.md:5]: Lol its when two devs meet, fall in love and run away together....  dev-elople-ment ;)

File: README.md (hunk 3 above)
Comment (Ian-Nicholls) [reply README.md:30]: ah yes so it is!

File: README.md (hunk 2 above)
Comment (Ian-Nicholls) [reply README.md:7]: Sure

**Review by joelanman — APPROVED** [review]
thanks! important stuff to reduce friction!

**Pull request merged**